pages.ja/common/aws-configure.md

```diff
@@ -0,0 +1,36 @@
+# aws configure
+
+> AWS CLI の設定を管理します。
+> 詳しくはこちら: <https://docs.aws.amazon.com/cli/latest/reference/configure/>
+
+- 対話形式で AWS CLI を設定する（新しい設定の作成、または既定の更新）:
+
+`aws configure`
+
+- 対話形式で AWS CLI の名前付きプロファイルを設定する（新規プロファイルの作成、または既存プロファイルの更新）:
+
+`aws configure --profile {{profile_name}}`
+
+- 特定の設定変数の値を表示する:
+
+`aws configure get {{name}}`
```

Comment:
```suggestion
`aws configure get {{名前}}`
```

Translation is missing for this placeholder. Google translation suggestion maybe have to be applied.

Reply:
@EmilySeville7cfg
Thanks for the review!
It looks good to me, so I'll adopt it.